### DISCUSSION TITLE
unable to move NSM dir to new block device because NSM is busy and cannot stop all services

### DISCUSSION THREAD

**Discussion by geckotechcr** [Security-Onion-Solutions/securityonion#5457] [Unsupported Versions] [answered]
on the new SecOnion 2.3.7 we have a heavynode that we wish to stop all services on so we can move our NSM mount to a new device.  However the old so-stop used to stop everything, but now it only stops one service at a time.
What is the optimal way to stop all services on HeavyNode (short of shutdown -now) so that we can move the NSM mount to a new block device?

**Comment by TOoSmOotH** [comment, marked as answer]
Disable the salt minion service and reboot. Make your changes and the enable the agent and reboot.

**Reply by geckotechcr** [reply]
Mike, thanks.   We had actually done this the first time, and not quite sure why it didn't work for us.   So we simply did a lazy unmount and did a dirty fix..   Next time we will pay more attention to ensure we DID disable the minion service to make sure SO doesnot regrab the NSM partition on reboot.

**Comment by dougburks** [comment]
Also see https://docs.securityonion.net/en/2.3/new-disk.html

**Reply by geckotechcr** [reply]
Doug, Thanks for the reply and pointer.  We had been using that link as the guide, which infact reinforces Mike's guidance on disabling minion.  As I told mike we must have done something wrong as that is what we had initially attempted.  We will be more careful next time to ensure minion was properly disabled pior to boot and ensuring the minion service is not active after reboot.

On another note: There used to be a so command that allowed stopping ALL services with a single command.  The new command seems to support stopping a single service at a time.  Is stopping all services in a single fell swoop frowned upon?

Thanks
Sean
